### ISSUE TITLE
Pretty slow; why not WebAssembly?

### ISSUE THREAD

**Issue by bryc** [kbrsh/slashjs#1]
Seems this is very slow due to 64-bit math emulation.

Using WebAssembly might be 90% faster due to 64-bit math compatibility

**Comment by kbrsh** [comment]
Yeah, I think it can be ported to WebAssembly for performance. I'd happily accept any PRs doing so. Check out the [reference implementation](https://github.com/kbrsh/slash) written in Assembly.
